docs/introduction.md

```diff
@@ -67,83 +96,207 @@ Flow is based upon the following set of rules:
 - Honest Stake Fraction
     - Flow requires more than two thirds (2/3) of [stake](#staking) from [collection](#collection-nodes), [consensus](#consensus-nodes) and [verification](#verification-nodes) nodes to be controlled by honest actors (for each node role separately). A super-majority (two thirds) of honest nodes probabilistically guarantees the safety of the Flow protocol.
 
-### Glossary
 
-#### Nodes
+### Errors
 
-<img alt="node roles" src="https://assets.website-files.com/5f6294c0c7a8cdd643b1c820/5fcff1a16213f9d33a6db5ff_ezgif.com-resize.gif" />
-
-##### Execution Nodes
-
-The execution nodes determine the results of [transactions](#transactions) when executed in the order determined by the [consensus nodes](#consensus-nodes).
-They are responsible for scaling the computational power of the blockchain, and are the only node role to have access to the **execution state**.
-They produce cryptographic attestations declaring the result of their efforts in the form of **execution receipts**.
-The receipts can be used to challenge the claims of an execution node when they are shown to be incorrect.
-They are also used to create proofs of the current state of the chain once they are known to be correct.
-In order to allow [verification nodes](#verification-nodes) to process the execution receipts, computations of a block are split into chunks.
-Each execution node publishes additional information about each chunk in its execution receipt for the block executed.
-Each chunk corresponds to a [collection](#collection-nodes).
-
-Errors introduced by the execution nodes are always guaranteed to have four critical attributes:
+Flow is designed to guarantee that any error introduced by the execution nodes are always guaranteed to have four critical attributes:
```

Comment:
I think this is true not just for execution nodes, but for all of them. You might want to pro-actively double check with the Flow team :)

Reply:
I also think that it's true for all errors. For instance, if there's a missing collection, execution nodes can issue a slashing challenge against the cluster of collection nodes that should provide the collection (the guarantors). That somewhat implies that the rules are the same.

However I think in the technical docs they were only explicit about the execution nodes. I will double check :)